### ISSUE TITLE
Sprint 40 (2/15/24-2/28/24)

### ISSUE THREAD

**Issue by lmorris3** [department-of-veterans-affairs/va.gov-team#76279]
## PTO

Holiday: Monday, Feb 19
2/22 - Phil AFK all day
2/20-2/26 - Rahul getting hitched
2/26 - Mateo OOO

## Sprint 40 Goals

**Campaigns**
_High-level goal: Launch behavior change communications campaign._ 

Engineering:
- USiP PR, launch (blocked until SSiT OCTO gives word)

Design/Research:
- Deploy GovDelivery segmented emails (blocked, may unblock this sprint)

Data:
- Monitor sign in and call center data (blocked until campaigns resume)
- Automate MPI, SSOe, MHV data ingest
- finish source account creation data view

Product:
- draft Identity roadmap with Tom

**Stretch Goals**

**IPP**
_High level goal: Prepare for IPP Pilot_

Engineering:
- Complete Login.gov IdP integration for Veteran intake
- Dockerize IPP applications
- Develop sign in page for Veteran intake app 
- Case number lookup from PA flow
- Prep environment for Proofing Agent Usability Testing (if needed)
- Onboarding to VA Datadog (for Logging, monitoring, observability)
- Start on LHDI security remediations

Research/Design:
- Pilot planning: identify pilot site
- Draft PA training "curriculum"
- Design improvements to Veteran intake 'home' screen
- Design a final screen for Veteran Intake flow

Product:
- Draft DIRA Statement (DIRA not needed bc MPI is authoritative source), review with Tom
- revise IPP charter to show vision of an in-person proofing service with VA
- IPP Timeline and budget materials - share with Tom

**Stretch Goals**
- Login.gov IdP Integration for Proofing Agent
- Develop sign in page for proofing agent app 
- Complete LHDI onboarding (team)
- VA Notify integration discovery
- Complete SME Interview synthesis
- Design A11y improvements to Veteran app

## What we want to demo at the end of the sprint

- IPP Pilot orientation and pilot materials
- IPP Veteran Intake Home Screen revisions
- LHDI CaTO Assessor
- Veteran sign in screen and IdP full integration (intake)
